docroot/themes/humsci/humsci_basic/src/scss/tools/_mixins.heros.scss

```diff
@@ -359,13 +359,18 @@
 }
 
 @mixin hb-hero-text-animation {
-  .hb-has-animation-enhancements & {
+  .hb-has-animation-enhancements.js & {
     opacity: 0;
   }
 
   @include hb-traditional {
     &.animate {
-      .hb-has-animation-enhancements & {
+      .hb-has-animation-enhancements.js .paragraph--type--hs-gradient-hero-slider .slick-active &,
+      .hb-has-animation-enhancements.js .paragraph--type--hs-gradient-hero-slider .unslick &,
+      .hb-has-animation-enhancements.js .ptype-hs-hero-image &,
+      .hb-has-animation-enhancements.js .field-hs-page-hero > :not(.paragraph--type--hs-carousel) .hb-hero-overlay &,
+      .hb-has-animation-enhancements.js .field-hs-page-hero > :not(.paragraph--type--hs-carousel) .slick-active &,
+      .hb-has-animation-enhancements.js .field-hs-page-hero > :not(.paragraph--type--hs-carousel) .unslick & {
```

Comment:
Unfortunately, this is what we have to do unless we refactor the hero-text mixin and hero components